build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+
+## typescript compile
+tsc
```

Comment:
```suggestion
npx tsc -p .
```

this way it will use the typescript version in node_modules instead of whatever the user has installed